guides/source/active_record_encryption.md

```diff
@@ -134,15 +134,15 @@ end
 
 You might need to ignore case when querying deterministically encrypted data. There are two options that can help you here.
 
-You can use the option `:downcase`  when declaring the encrypted attribute. This will make that content is downcased before being encrypted.
+You can use the option `:downcase` when declaring the encrypted attribute. This will downcase that content before encrypting it.
 
 ```ruby
 class Person
   encrypts :email_address, deterministic: true, downcase: true
 end
 ```
 
-When using `:downcase` the original case is lost. There might be cases where you need to preserve the original case when reading the value, but you need to ignore the case when querying. For those cases you can use the option `:ignore_case` which requires you to add a new column named `original_<column_name>` to store the content with the case unchanged:
+When using `:downcase` the original case is lost. There might be situations where you need to preserve the original case when reading the value, but you need to ignore the case when querying. For those situations you can use the option `:ignore_case` which requires you to add a new column named `original_<column_name>` to store the content with the case unchanged:
```

Comment:
```suggestion
When using `:downcase` the original case is lost. In some situations, you might need to preserve the original case, but ignore that case when querying. For those situations, you can use the `:ignore_case` option which requires a new column named `original_<column_name>` to store the content with the case unchanged:
```

I think this can also be shortened a bit.